zed_wrapper/src/nodelet/src/zed_wrapper_nodelet.cpp

```diff
@@ -3004,5 +3030,9 @@ namespace zed_wrapper {
         mZed.setCameraSettings(sl::CAMERA_SETTINGS_LED_STATUS, new_status);
 
         return (new_status == 1);
+#else
+        ROS_WARN("Led control is available with SDK v2.8 or major");
```

Comment:
"LED control requires the ZED SDK v2.8 or newer"

Reply:
Done